receiver/k8sclusterreceiver/internal/collection/collector.go

```diff
@@ -78,8 +82,42 @@ type DataCollector struct {
 	allocatableTypesToReport []string
 }
 
+var reportCPUMetricsAsDoubleFeatureGate = featuregate.Gate{
+	ID:      reportCPUMetricsAsDoubleFeatureGateID,
+	Enabled: false,
+	Description: "The k8s container and node cpu metrics being reported by the k8sclusterreceiver are transitioning " +
+		"from being reported as integer millicpu units to being reported as double cpu units to adhere to " +
+		"opentelemetry cpu metric specifications. Please update any monitoring this might affect, the change will " +
+		"cause cpu metrics to be double instead of integer values as well as metric values will be scaled down by " +
+		"1000x. You can control whether the k8sclusterreceiver reports container and node cpu metrics in double cpu " +
+		"units instead of integer millicpu units with the feature gate listed below. Note, feature gate identifiers " +
+		"prefixed with - will disable the gate and prefixing with + or with no prefix will enable the gate.\n" +
+		"Feature gate name:\n" +
+		"\treceiver.k8sclusterreceiver.reportcpumetricsasdouble\n" +
+		"Start the otelcol with the feature gate enabled:\n" +
+		"\totelcol {other_arguments} --feature-gates=receiver.k8sclusterreceiver.reportcpumetricsasdouble\n" +
+		"Start the otelcol with the feature gate disabled:\n" +
+		"\totelcol {other_arguments} --feature-gates=-receiver.k8sclusterreceiver.reportcpumetricsasdouble\n" +
+		"For more details see:\n" +
+		"\thttps://github.com/open-telemetry/opentelemetry-collector-contrib/issues/8115\n" +
+		"\thttps://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemcpu---processor-metrics\n",
+}
+
+func init() {
+	featuregate.Register(reportCPUMetricsAsDoubleFeatureGate)
+}
+
 // NewDataCollector returns a DataCollector.
 func NewDataCollector(logger *zap.Logger, nodeConditionsToReport, allocatableTypesToReport []string) *DataCollector {
+	logger.Info("WARNING - Breaking Change: " + reportCPUMetricsAsDoubleFeatureGate.Description)
```

Comment:
I think throwing this big amount of documentation here is lot. Probably better to keep the warning short and give a link to the documentation instead. And probably warning is not needed if `featuregate.IsEnabled`, just a one line confirmation.

Reply:
Got it. Updated.